diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/configure.in openCryptoki-2.1.5-6_64_fix/configure.in
--- openCryptoki-2.1.5-6/configure.in	2004-03-03 11:55:06.000000000 -0600
+++ openCryptoki-2.1.5-6_64_fix/configure.in	1986-08-03 04:53:29.639272160 -0500
@@ -8,8 +8,8 @@
 dnl Checks for libraries.
 
 dnl Checks for header files.
-AC_DISABLE_STATIC
-AC_PROG_LIBTOOL
+#AC_DISABLE_STATIC
+#AC_PROG_LIBTOOL
 
 AC_HEADER_STDC
 AC_CHECK_HEADERS(fcntl.h limits.h strings.h sys/file.h syslog.h unistd.h)
@@ -172,14 +172,14 @@
 AC_PROG_CC
 
 # Override CC iff we're on a ppc64 to cross compile the 64bit shared objects
-if [[[ $arch == ppc64 ]]]; then
-	if [[[ -x /opt/cross/bin/powerpc64-linux-gcc ]]]; then
-		CC=/opt/cross/bin/powerpc64-linux-gcc
-	else
-		AC_MSG_RESULT([*** /opt/cross/bin/powerpc64-linux-gcc NOT FOUND ***])
-		AC_MSG_RESULT([*** 64bit cross compiler was not found, please install the cross-ppc64-* packages ***])
-	fi
-fi
+#if [[[ $arch == ppc64 ]]]; then
+#	if [[[ -x /opt/cross/bin/powerpc64-linux-gcc ]]]; then
+#		CC=/opt/cross/bin/powerpc64-linux-gcc
+#	else
+#		AC_MSG_RESULT([*** /opt/cross/bin/powerpc64-linux-gcc NOT FOUND ***])
+#		AC_MSG_RESULT([*** 64bit cross compiler was not found, please install the cross-ppc64-* packages ***])
+#	fi
+#fi
 
 AC_OUTPUT(Makefile usr/Makefile \
           usr/lib/Makefile \
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/README openCryptoki-2.1.5-6_64_fix/README
--- openCryptoki-2.1.5-6/README	2004-03-03 15:05:47.000000000 -0600
+++ openCryptoki-2.1.5-6_64_fix/README	1986-08-02 12:59:38.499013000 -0500
@@ -65,8 +65,8 @@
   On 64bit s390x, a 64bit gcc is used to compile the only 2 64bit objects
 used in openCryptoki:
 
-  /usr/lib/pkcs11/PKCS11_API.so64		64bit shared library
-  /usr/lib/pkcs11/stdll/PKCS11_ICA.so64		64bit shared library
+  /usr/lib64/pkcs11/PKCS11_API.so		64bit shared library
+  /usr/lib64/pkcs11/stdll/PKCS11_ICA.so		64bit shared library
 
   On ppc64, the 64bit cross compiler is used to compile these 2 objects. On
 both s390x and ppc64, the 32bit libraries and executables are installed as
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/rpm/openCryptoki.spec openCryptoki-2.1.5-6_64_fix/rpm/openCryptoki.spec
--- openCryptoki-2.1.5-6/rpm/openCryptoki.spec	2004-03-10 09:02:01.000000000 -0600
+++ openCryptoki-2.1.5-6_64_fix/rpm/openCryptoki.spec	1986-08-02 12:59:38.520975000 -0500
@@ -107,12 +107,9 @@
 }
 #./autoversion
 #autoreconf --force --install
-# openCryptoki appends a 'so64' to 64bit shared objects and they stay
-# in /usr/lib, so we leave libdir allone.  libexecdir is not needed as
-# well, since IBM does not support the 64bit tools, if a 32bit
-# compatibility exists in the 64bit architecture, so no conflicts
-# possible here as well.
-#./configure # --libdir=%_libdir
+%ifarch %openCryptoki_64bit_arch
+./configure --libdir=/usr/lib64
+%endif
 #/bin/pwd
 #/usr/bin/aclocal
 #/usr/bin/automake
@@ -190,7 +187,7 @@
 %else # not openCryptoki_32bit_arch  but  64bit arch
 
 %files 64bit
-/usr/lib/pkcs11/PKCS11_API.so64
-/usr/lib/pkcs11/stdll/PKCS11_ICA.so64
+/usr/lib64/pkcs11/PKCS11_API.so
+/usr/lib64/pkcs11/stdll/PKCS11_ICA.so
 %endif
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/driver/driver.c openCryptoki-2.1.5-6_64_fix/testcases/driver/driver.c
--- openCryptoki-2.1.5-6/testcases/driver/driver.c	2003-06-19 15:56:39.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/driver/driver.c	1986-08-03 09:43:58.264201440 -0500
@@ -243,7 +243,7 @@
    CK_RV  (*pfoo)();
    void    *d;
    char    *e;
-   char    *f = "/usr/lib/pkcs11/PKCS11_API.so";
+   char    *f = "PKCS11_API.so";
 
 #endif
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/init_tok/init_tok.c openCryptoki-2.1.5-6_64_fix/testcases/init_tok/init_tok.c
--- openCryptoki-2.1.5-6/testcases/init_tok/init_tok.c	2003-12-15 13:20:13.000000000 -0600
+++ openCryptoki-2.1.5-6_64_fix/testcases/init_tok/init_tok.c	1986-08-03 09:09:35.783234272 -0500
@@ -140,7 +140,8 @@
    CK_RV  (*pfoo)();
    void    *d;
    char    *e;
-   char		*x="/usr/lib/pkcs11/PKCS11_API.so";
+//   char		*x="/usr/lib/pkcs11/PKCS11_API.so";
+   char		*x="PKCS11_API.so";
 
 #endif
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/mkobj/mkobj.c openCryptoki-2.1.5-6_64_fix/testcases/mkobj/mkobj.c
--- openCryptoki-2.1.5-6/testcases/mkobj/mkobj.c	2002-08-01 13:00:21.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/mkobj/mkobj.c	1986-08-03 09:47:51.170204936 -0500
@@ -139,7 +139,7 @@
    CK_RV  (*pfoo)();
    void    *d;
    char    *e;
-   char 	*z="/usr/lib/pkcs11/PKCS11_API.so";
+   char 	*z="PKCS11_API.so";
 #endif
 
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/oc-digest/oc-digest.c openCryptoki-2.1.5-6_64_fix/testcases/oc-digest/oc-digest.c
--- openCryptoki-2.1.5-6/testcases/oc-digest/oc-digest.c	2003-07-14 13:32:17.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/oc-digest/oc-digest.c	1986-08-03 09:44:32.724288776 -0500
@@ -287,7 +287,7 @@
 
 int do_GetFunctionList(void)
 {
-	char *pkcslib = "/usr/lib/pkcs11/PKCS11_API.so";
+	char *pkcslib = "PKCS11_API.so";
 	CK_RV (*func_ptr)();
 	int rc;
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/rsa_keygen/rsa_keygen.c openCryptoki-2.1.5-6_64_fix/testcases/rsa_keygen/rsa_keygen.c
--- openCryptoki-2.1.5-6/testcases/rsa_keygen/rsa_keygen.c	2002-08-01 13:00:21.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/rsa_keygen/rsa_keygen.c	1986-08-03 09:45:36.260214448 -0500
@@ -310,7 +310,7 @@
    CK_RV  (*pfoo)();
    void    *d;
    char    *e;
-   char    default_lib[]="/usr/lib/pkcs11/PKCS11_API.so";
+   char    default_lib[]="PKCS11_API.so";
 
 #endif
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/rsa_test/rsa_test.c openCryptoki-2.1.5-6_64_fix/testcases/rsa_test/rsa_test.c
--- openCryptoki-2.1.5-6/testcases/rsa_test/rsa_test.c	2002-08-01 13:00:21.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/rsa_test/rsa_test.c	1986-08-03 09:48:11.943242872 -0500
@@ -521,7 +521,7 @@
    CK_RV  (*pfoo)();
    void    *d;
    char    *e;
-   char	   f[]="/usr/lib/pkcs11/PKCS11_API.so";
+   char	   f[]="PKCS11_API.so";
 
 #endif
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/rsa_test/rsa_test_genkey.c openCryptoki-2.1.5-6_64_fix/testcases/rsa_test/rsa_test_genkey.c
--- openCryptoki-2.1.5-6/testcases/rsa_test/rsa_test_genkey.c	2002-08-01 13:00:21.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/rsa_test/rsa_test_genkey.c	1986-08-03 09:48:21.931187608 -0500
@@ -522,7 +522,7 @@
    CK_RV  (*pfoo)();
    void    *d;
    char    *e;
-   char	   f[]="/usr/lib/pkcs11/PKCS11_API.so";
+   char	   f[]="PKCS11_API.so";
 
 #endif
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/rsa_test/rsa_thread_test.c openCryptoki-2.1.5-6_64_fix/testcases/rsa_test/rsa_thread_test.c
--- openCryptoki-2.1.5-6/testcases/rsa_test/rsa_thread_test.c	2002-08-01 13:00:21.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/rsa_test/rsa_thread_test.c	1986-08-03 09:48:31.060265216 -0500
@@ -624,7 +624,7 @@
    CK_RV  (*pfoo)();
    void    *d;
    char    *e;
-   char	   f[]="/usr/lib/pkcs11/PKCS11_API.so";
+   char	   f[]="PKCS11_API.so";
 
 #endif
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/speed/speed.c openCryptoki-2.1.5-6_64_fix/testcases/speed/speed.c
--- openCryptoki-2.1.5-6/testcases/speed/speed.c	2002-08-01 13:00:21.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/speed/speed.c	1986-08-03 09:46:07.648234272 -0500
@@ -277,7 +277,7 @@
    CK_RV  (*pfoo)();
    void    *d;
    char    *e;
-   char    *f="/usr/lib/pkcs11/PKCS11_API.so";
+   char    *f="PKCS11_API.so";
 #endif
 
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/threadmkobj/threadmkobj.c openCryptoki-2.1.5-6_64_fix/testcases/threadmkobj/threadmkobj.c
--- openCryptoki-2.1.5-6/testcases/threadmkobj/threadmkobj.c	2002-08-01 13:00:21.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/threadmkobj/threadmkobj.c	1986-08-03 09:46:24.960264976 -0500
@@ -144,7 +144,7 @@
    CK_RV  (*pfoo)();
    void    *d;
    char    *e;
-   char 	*z="/usr/lib/pkcs11/PKCS11_API.so";
+   char 	*z="PKCS11_API.so";
 #endif
 
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/tok_obj/tok_obj.c openCryptoki-2.1.5-6_64_fix/testcases/tok_obj/tok_obj.c
--- openCryptoki-2.1.5-6/testcases/tok_obj/tok_obj.c	2002-08-01 13:00:21.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/tok_obj/tok_obj.c	1986-08-03 09:48:53.270215880 -0500
@@ -139,7 +139,7 @@
    CK_RV  (*pfoo)();
    void    *d;
    char    *e;
-   char    *f = "/usr/lib/pkcs11/PKCS11_API.so";
+   char    *f = "PKCS11_API.so";
 
 #endif
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/v2.11/aes_func.c openCryptoki-2.1.5-6_64_fix/testcases/v2.11/aes_func.c
--- openCryptoki-2.1.5-6/testcases/v2.11/aes_func.c	2003-05-02 11:38:53.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/v2.11/aes_func.c	1986-08-03 09:46:57.112170672 -0500
@@ -1782,7 +1782,7 @@
 
 int do_GetFunctionList(void)
 {
-        char *pkcslib = "/usr/lib/pkcs11/PKCS11_API.so";
+        char *pkcslib = "PKCS11_API.so";
         CK_RV (*func_ptr)();
         int rc;
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/v2.11/hw_fn.c openCryptoki-2.1.5-6_64_fix/testcases/v2.11/hw_fn.c
--- openCryptoki-2.1.5-6/testcases/v2.11/hw_fn.c	2003-05-02 11:38:54.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/v2.11/hw_fn.c	1986-08-03 09:47:14.925257376 -0500
@@ -351,7 +351,7 @@
 
 int do_GetFunctionList(void)
 {
-	char *pkcslib = "/usr/lib/pkcs11/PKCS11_API.so";
+	char *pkcslib = "PKCS11_API.so";
 	CK_RV (*func_ptr)();
 	int rc;
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/v2.11/login.c openCryptoki-2.1.5-6_64_fix/testcases/v2.11/login.c
--- openCryptoki-2.1.5-6/testcases/v2.11/login.c	2003-05-02 11:38:54.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/v2.11/login.c	1986-08-03 09:47:25.316275856 -0500
@@ -306,7 +306,7 @@
 
 int do_GetFunctionList(void)
 {
-	char *pkcslib = "/usr/lib/pkcs11/PKCS11_API.so";
+	char *pkcslib = "PKCS11_API.so";
 	CK_RV (*func_ptr)();
 	int rc;
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/testcases/v2.11/rijndael.c openCryptoki-2.1.5-6_64_fix/testcases/v2.11/rijndael.c
--- openCryptoki-2.1.5-6/testcases/v2.11/rijndael.c	2003-05-02 11:38:54.000000000 -0500
+++ openCryptoki-2.1.5-6_64_fix/testcases/v2.11/rijndael.c	1986-08-03 09:47:35.310219680 -0500
@@ -1017,7 +1017,7 @@
 
 int do_GetFunctionList(void)
 {
-	char *pkcslib = "/usr/lib/pkcs11/PKCS11_API.so";
+	char *pkcslib = "PKCS11_API.so";
 	CK_RV (*func_ptr)();
 	int rc;
 
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/usr/lib/pkcs11/api/Makefile.am openCryptoki-2.1.5-6_64_fix/usr/lib/pkcs11/api/Makefile.am
--- openCryptoki-2.1.5-6/usr/lib/pkcs11/api/Makefile.am	2004-02-10 10:55:12.000000000 -0600
+++ openCryptoki-2.1.5-6_64_fix/usr/lib/pkcs11/api/Makefile.am	1986-08-03 09:18:25.648225456 -0500
@@ -7,9 +7,10 @@
 
 install-data-local:
 	mkdir -p $(INSROOT)/usr/lib/pkcs11/stdll; \
+	mkdir -p $(INSROOT)/usr/lib64/pkcs11/stdll; \
 	if [ $(shell uname -m) == s390x -o $(shell uname -m) == ppc64 ]; then \
-		cp PKCS11_API.so $(INSROOT)/usr/lib/pkcs11/PKCS11_API.so64; \
+		cp PKCS11_API.so $(INSROOT)/usr/lib64/pkcs11/PKCS11_API.so; ln -s $(INSROOT)/usr/lib64/pkcs11/PKCS11_API.so $(INSROOT)/usr/lib64/; \
 	else \
-		cp PKCS11_API.so $(INSROOT)/usr/lib/pkcs11/; \
+		cp PKCS11_API.so $(INSROOT)/usr/lib/pkcs11/; ln -s $(INSROOT)/usr/lib/pkcs11/ $(INSROOT)/usr/lib/ ; \
 	fi; \
 	rm -f /tmp/PKCS11_API.*;
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/usr/lib/pkcs11/ica_s390_stdll/Makefile.am openCryptoki-2.1.5-6_64_fix/usr/lib/pkcs11/ica_s390_stdll/Makefile.am
--- openCryptoki-2.1.5-6/usr/lib/pkcs11/ica_s390_stdll/Makefile.am	2004-02-10 10:55:12.000000000 -0600
+++ openCryptoki-2.1.5-6_64_fix/usr/lib/pkcs11/ica_s390_stdll/Makefile.am	1986-08-02 12:59:38.823677000 -0500
@@ -14,6 +14,6 @@
 
 install-data-local:
 	mkdir -p $(INSROOT)/usr/lib/pkcs11/stdll; \
- 	if [ $(shell uname -m) == s390x ] ;then cp PKCS11_ICA.so $(INSROOT)/usr/lib/pkcs11/stdll/PKCS11_ICA.so64; \
+	if [ $(shell uname -m) == s390x ] ;then cp PKCS11_ICA.so $(INSROOT)/usr/lib64/pkcs11/stdll/PKCS11_ICA.so; ln -s $(INSROOT)/usr/lib64/pkcs11/stdll/PKCS11_ICA.so /usr/lib/pkcs11/stdll/PKCS11_ICA.so64; \
 	else cp PKCS11_ICA.so $(INSROOT)/usr/lib/pkcs11/stdll/; fi; \
 	rm -f /tmp/PKCS11_ICA.*;
diff -Naur -x configure -x pkcsslotd -x pkcsconf -x pkcs_slot -x '#Makefile.am#' -x out.txt -x Makefile.in -x Makefile -x config.status -x config.log -x autom4te.cache -x CVS -x '*~' -x '*o' openCryptoki-2.1.5-6/usr/lib/pkcs11/ica_stdll/Makefile.am openCryptoki-2.1.5-6_64_fix/usr/lib/pkcs11/ica_stdll/Makefile.am
--- openCryptoki-2.1.5-6/usr/lib/pkcs11/ica_stdll/Makefile.am	2004-02-10 10:55:11.000000000 -0600
+++ openCryptoki-2.1.5-6_64_fix/usr/lib/pkcs11/ica_stdll/Makefile.am	1986-08-03 09:16:57.281214752 -0500
@@ -14,9 +14,9 @@
 install-data-local:
 	mkdir -p $(INSROOT)/usr/lib/pkcs11/stdll; \
 	if [ $(shell uname -m) == ppc64 ]; then \
-		cp PKCS11_ICA.so $(INSROOT)/usr/lib/pkcs11/stdll/PKCS11_ICA.so64; \
+		cp PKCS11_ICA.so $(INSROOT)/usr/lib64/pkcs11/stdll/PKCS11_ICA.so; ln -s $(INSROOT)/usr/lib64/pkcs11/stdll/PKCS11_ICA.so $(INSROOT)/usr/lib/pkcs11/stdll/PKCS11_ICA.so64; ln -s $(INSROOT)/usr/lib64/pkcs11/stdll/PKCS11_ICA.so $(INSROOT)/usr/lib64/; \
 	elif [ $(shell uname -m) != s390x ]; then \
-		cp PKCS11_ICA.so $(INSROOT)/usr/lib/pkcs11/stdll/PKCS11_ICA.so; \
+		cp PKCS11_ICA.so $(INSROOT)/usr/lib/pkcs11/stdll/PKCS11_ICA.so; ln -s $(INSROOT)/usr/lib/pkcs11/stdll/PKCS11_ICA.so $(INSROOT)/usr/lib/; \
 	fi; \
 	rm -f /tmp/PKCS11_ICA.*;
 
